npc/quests/bard_quest.txt

```diff
@@ -1923,12 +1923,12 @@ morocc,134,111,3	script	Bard#3	2_M_BARD_ORIENT,{
 				mes "There's no reason to be scared. The scary people with swords only use them on monsters, okay?";
 			} else if (BaseClass == Job_Acolyte) {
 				mes "You don't need to be scared, that person is a servant of God, okay?";
-			} else if (Class == Job_Thief || Class == Job_Rogue) {
+			} else if (BaseJob == Job_Thief || BaseJob == Job_Rogue) {
 				mes "There's no reason to be afraid of this riffraff, your Uncle Kino is here, okay?";
-			} else if (BaseClass == Job_Assassin) {
+			} else if (BaseJob == Job_Assassin) {
 				mes "There's no reason to be afraid. I know that person looks scary, but you're a good girl, so you'll be okay.";
-			} else if (BaseClass == Job_Blacksmith) {
-				mes "There's no reason to be scared, honey. It's just a Blacksmith.";
+			} else if (BaseJob == Job_Blacksmith) {
+				mesf("There's no reason to be scared, honey. It's just a %s.", jobname(Class));
```

Comment:
`@job 4064`
returns 
There's no reason to be scared, honey. It's just a Mechanic T.

com'on no need to change that Blacksmith, right ???

but this is just a flavor text so don't want to argue about cosmetic stuffs

Reply:
Someone has to fix that in a separate PR then ;)